### PR TITLE
Magic Eight Ball - Marjaana 

### PR DESCRIPTION
**Pull request by MayaBlixt**
My reflections on how this week's project turned out:

Things I'd like to have clarified or explained in more detail:

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -1,26 +1,26 @@
-import React from 'react'
-import styled from 'styled-components/native'
+import React, { useState } from "react";
+import styled from "styled-components/native";
+import { NavigationContainer } from "@react-navigation/native";
+import { createStackNavigator } from "@react-navigation/stack";
+import HomeScreen from "./components/HomeScreen";
```
Comment (EmOlsz) [inline App.js:5]: Let's remember about outer and local dependencies line break

File: components/BallScreen.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        event.preventDefault()
+        fetch(uri)
+            .then(response => response.json())
+            .then(json => {
+                setAnswer(json.magic.answer);
+            });
+    };
+
+  return ( 
+ <ScreenContainer source={backgroundImage}>
```
Comment (EmOlsz) [inline components/BallScreen.js:25]: Wrong indentations

File: components/BallScreen.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+      <BallTitle> What would you like to ask ? </BallTitle>
+        <BallTextDescription> 
+            Enter your question below
+        </BallTextDescription>
+        <BallTextInput
+            onChangeText={text => setQuestion(text)}
```
Comment (EmOlsz) [inline components/BallScreen.js:31]: Or just `onChangeText={setQuestion}`

File: styledComponents/ScreenStyling.js [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+  margin-top: 100px;
+  flex: 5;
+`;
+
+export const ScreenButton = styled.TouchableOpacity`
+  letter-spacing: 3px;
+  alignItems: center;
+  backgroundColor: #f4f4f4;
+  padding: 20px;
+  margin: 0px 20px;
```
Comment (EmOlsz) [inline styledComponents/ScreenStyling.js:45]: We don't need to specify unit, if value is equal 0 - `margin: 0 20px;`

File: styledComponents/ScreenStyling.js [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+  text-transform: uppercase;
+  padding: 0 20px;
+  color:#524f4f;
+`;
+
+export const BallTextInput = styled.TextInput`
+    height: 50px; 
+    width: 350px;
+    borderColor: gray;
+    backgroundColor: #f4f4f4;
```
Comment (EmOlsz) [inline styledComponents/ScreenStyling.js:62]: In styled components we define styles in kebab-case, not camelCase

**Review by EmOlsz — CHANGES_REQUESTED** [review]

File: App.js [hunk 6]
```diff
@@ -1,26 +1,26 @@
...
-    <Container>
-      <Title>This is your cool app!</Title>
-      <Title>Go to App.js and start coding</Title>
-      <Title>💅💅💅</Title>
-    </Container>
-  )
-}
-
-export default App
+    <NavigationContainer>
```
Comment (EmOlsz) [inline App.js:14]: Great job with Navigation :)

File: App.js (hunk 1 above)
Comment (MayaBlixt) [reply App.js:7]: That is right! Added a line break :)

File: components/BallScreen.js (hunk 2 above)
Comment (MayaBlixt) [reply components/BallScreen.js]: Updated the indentations.

File: components/BallScreen.js (hunk 3 above)
Comment (MayaBlixt) [reply components/BallScreen.js]: Nice! Changed to onChangeText={setQuestion}

File: styledComponents/ScreenStyling.js (hunk 4 above)
Comment (MayaBlixt) [reply styledComponents/ScreenStyling.js]: Removed px from 0px.

File: styledComponents/ScreenStyling.js (hunk 5 above)
Comment (MayaBlixt) [reply styledComponents/ScreenStyling.js]: Changed to kebab-case :)

File: App.js (hunk 6 above)
Comment (MayaBlixt) [reply App.js:16]: 😸

**Comment by MayaBlixt** [comment]
Made asked changes and pushed code.

**Review by EmOlsz — APPROVED** [review]

**Pull request closed without merging**